test/extended/util/prometheus/helpers.go

```diff
@@ -444,3 +498,12 @@ func ForEachAlertingRule(rules map[string][]promv1.AlertingRule, f func(a promv1
 
 	return fmt.Errorf("Incompliant rules detected:\n\n%s", strings.Join(allViolations.List(), "\n"))
 }
+
+// url.JoinPath but with no error checks
+func JoinUrl(base string, paths ...string) string {
```

Comment:
(nit)
```suggestion
// MustJoinUrlPath behaves like url.JoinPath but it will panic in case of error.
func MustJoinUrlPath(base string, paths ...string) string {
```

```suggestion
func MustJoinUrlPath(base string, paths ...string) string {
```